timetracker/entries/templates/projects.html

```diff
@@ -12,8 +12,17 @@
   <ul class="list-group col-md-3">
     {% for project in project_list %}
       <li class="list-group-item">
-        <a href="{% url 'project-detail' pk=project.pk %}">{{ project.name }}</a> (<a href="{% url 'client-detail' pk=project.client.pk %}">{{ project.client.name }}</a>)
-        <a class="btn btn-warning btn-xs pull-right" href="{% url 'project-detail' pk=project.pk %}">Edit</a>
+        {% if project.author == request.user %}<a href="{% url 'project-detail' pk=project.pk %}">{% endif %}
```

Comment:
Ok. In this case we have 2 unique links on one line that can be "on on", "on off" or "off off" so there's going to be a bit of repetition. Ok to leave as it is or should we use your approach?